test/jwt.hs.tests.js

```diff
@@ -96,16 +96,26 @@ describe('HS256', function() {
     });
   });
 
-  describe('should fail verification gracefully with trailing space in the jwt', function() {
+  describe('should fail verification gracefully on malformed token', function() {
     var secret = 'shhhhhh';
     var token  = jwt.sign({ foo: 'bar' }, secret, { algorithm: 'HS256' });
 
-    it('should return the "invalid token" error', function(done) {
+    it('should return the "jwt malformed" error with a trailing space', function(done) {
```

Comment:
What might have happened to have the malformed JWT, it's because on my app, it has been working fine, and then lately I get `jwt malformed` error message, if I refresh the page, I will no longer see that error again, can you please give me details on this.